archinstall/lib/general.py

```diff
@@ -430,10 +430,15 @@ def create_session(self) -> bool:
 
 		return True
 
-	def decode(self, *args, **kwargs) -> Optional[str]:
-		if self.session:
-			return self.session._trace_log.decode(*args, **kwargs)
-		return None
+	def decode(self, encoding: str = 'utf-8', errors='backlashreplace', strip: bool = True) -> str:
```

Comment:
I think `errors` can be dropped if it is not used otherwise please fix the typo in 'backslashreplace'.

Reply:
I actually forgot to pass that into the `decode`